### PR TITLE
Some minor Ecobbomy items (Paystand Functions, Merchant Necklace for Admins)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
### Paystands
Paystands have some more functionality based on how you hit them with your registered card

- Help/Harm: Default (Lock it down)
- Disarm: Rename
- Grab: Replace "Pay as much as you want" with an amount you decide. 0 returns it to the old mode.

These options are shown on examine while having the card in your hand.

Paystands also had their lock message edited to clarify it's not the same as anchoring.

### Blingy Bling
Adds a merchant bling necklace that converts items to their export value in holochips (Admin-Only). Can also look at prices (post-fees only). Has a variable admins can edit `profit_scaling` to apply "processing fees".

## Why It's Good For The Game
Paystand Change: Allows for more customization of the paystand to make it more practical to users.

Merchant Necklace: Admin Event-Support

## Changelog
:cl: Ecobbomy
tweak: Paystands have a few new functions. Examine your paystand with the registered card in hand to view these options!
admin: there is a bling necklace subtype that allows you to sell items for credits, hassle-free (search "merchant" in the spawn menu)! Useful for merchant-based events.
/:cl:

### CODE REVIEW

**Comment by RaveRadbury** [comment]
Speedmerge

**Comment by bobbahbrown** [comment]
good pr

NOTICE: this is a sponsored message

File: code/modules/economy/pay_stand.dm [hunk 1]
```diff
@@ -106,7 +126,13 @@
...
+		return FALSE
+	. = ..()
+
+/obj/machinery/paystand/examine(mob/user)
 	. = ..()
-	
+	if(force_fee)
+		. += "<span class='warning'>This paystand forces a payment of <b>[force_fee]</b> credit\s per swipe instead of a variable amount.</span>"
+	if(user.get_active_held_item() == my_card)
+		. += "<span class='notice'>Paystands can be edited through swiping your card with different intents. <b>Disarm</b> allows editing the name while <b>Grab</b> changes payment functionality."
```
Comment (bobbahbrown) [inline code/modules/economy/pay_stand.dm]:
```suggestion
		. += "<span class='notice'>Paystands can be edited through swiping your card with different intents. <b>Disarm</b> allows editing the name while <b>Grab</b> changes payment functionality.</span>"
```

more span

**Comment by HomocidlePancakes** [comment]
Ecobbomy. Really? Is this how we call it now?

**Comment by Time-Green** [comment]
You should add a hint to the description or this will end up completely overlooked by most people

**Comment by ExcessiveUseOfCobblestone** [comment]
it does already but requires you to be holding the registered card (and also if the card is null you can see it when you're not holding anything).

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> Ecobbomy. Really? Is this how we call it now?

I was just making a funny, idc what you choose to call it.

File: code/modules/clothing/neck/_neck.dm [hunk 2]
```diff
@@ -194,6 +194,39 @@
 	icon = 'icons/obj/clothing/neck.dmi'
 	icon_state = "bling"
 
+/obj/item/clothing/neck/necklace/dope/merchant
+	desc = "Don't ask how it works, the proof is in the holochips!"
+	var/profit_scaling = 1 //in case you want to RP taxes or something
```
Comment (ninjanomnom) [inline code/modules/clothing/neck/_neck.dm]: Use doc comments

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
do you want autodoc for the datum's values as well?

**Comment by ninjanomnom** [comment]
Nah they just seemed like comments that were perfect for autodocs except they weren't in the right format

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**